Dockerfile

```diff
@@ -79,6 +81,7 @@ USER builder
 RUN --mount=source=.cargo,target=/home/builder/.cargo \
     --mount=type=cache,target=/home/builder/.cache,from=cache,source=/cache \
     --mount=type=cache,target=/home/builder/rpmbuild/BUILD/sources/models/src/variant,from=variantcache,source=/variantcache \
+    --mount=type=cache,target=/home/builder/rpmbuild/BUILD/sources/logdog/static,from=variantcache,source=/variantcache/logdog \
```

Comment:
I think we should consider doing `/variantcache/models` like you did with `/variantcache/logdog` so we don't have any potential interaction between the two crates' storage.

Reply:
[afdc876](https://github.com/bottlerocket-os/bottlerocket/compare/6eb611d17506ab4aad98e5db4cd17b00bacd0839..afdc8765a9e97bb1d4436eda9b0c964fce24ab95)